Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/InfiniteCanvas.cs

```diff
@@ -348,8 +351,51 @@ public void ImportFromJson(string json)
         }
 
         /// <summary>
-        /// This event triggered after each render happended because of any change in the canvas elements.
+        /// This method exports the max possible view of the InfiniteCanvas drawings as offScreen drawings that can be converted to image.
+        /// Max is calculated using CanvasDevice.MaximumBitmapSizeInPixels
+        /// </summary>
+        /// <param name="stream">The target stream.</param>
+        /// <param name="bitmapFileFormat">the specified format.</param>
+        public async Task SaveMaxViewAsync(IRandomAccessStream stream, BitmapFileFormat bitmapFileFormat)
+        {
+            var offScreen = _drawingSurfaceRenderer.ExportMaxOffScreenDrawings();
+            await offScreen.SaveAsync(stream, MapToCanvasBitmapFileFormat(bitmapFileFormat));
+        }
+
+        /// <summary>
+        /// This method exports the current view of the InfiniteCanvas drawings as offScreen drawings that can be converted to image.
+        /// </summary>
+        /// <param name="stream">The target stream.</param>
+        /// <param name="bitmapFileFormat">the specified format.</param>
+        public async Task SaveCurrentViewAsync(IRandomAccessStream stream, BitmapFileFormat bitmapFileFormat)
```

Comment:
Hey @IbraheemOsama, tested this out, this was my current view in the app:
![image](https://user-images.githubusercontent.com/24302614/52086879-d9a1f680-255c-11e9-9fd2-386cb93502d2.png)

But my exported image was this:
![testexportcurrent](https://user-images.githubusercontent.com/24302614/52086892-e3c3f500-255c-11e9-80e2-6c8a72a73eaa.png)

Which wasn't what I expected.  Is something wrong here or did I misunderstand?  This 'current' image actually had the gutter, where my max image had less space, which seemed odd too:

![textexportmax](https://user-images.githubusercontent.com/24302614/52086974-179f1a80-255d-11e9-8321-42e416d067fe.png)

Reply:
To clarify, I'd expect exporting the 'current' view to provide this image (without gutter scroll controls obviously :)):
![image](https://user-images.githubusercontent.com/24302614/52087138-7a90b180-255d-11e9-9cac-d92b5d949d81.png)

Reply:
Ah, I see.  I think changing the name would be good then.  But I don't know if I see the value in having two methods if it doesn't clip to the visible view.  Maybe we just export the full canvas for now and look at how we can crop it to the viewport for a 2nd method later?

Reply:
The method is removed :)